infinispan-embedded/src/test/java/org/hibernate/ogm/datastore/infinispan/test/dialect/impl/counter/GlobalConfigNotEnabledTableGenerationTest.java

```diff
@@ -97,23 +80,14 @@ public void testExceptionWithTableGenerator() throws Exception {
 
 		OgmEntityManagerFactory emFactory = ( (OgmEntityManagerFactory) getFactory() );
 		SessionFactoryImplementor sessionFactory = emFactory.getSessionFactory();
```

Comment:
This will need to be rebased after the merge of the ORM 5.2 branch. It should be easy, it's just the way the `sessionFactory` is obtained which was changed. Feel free to ping me if you have any issue with it.